spec/spec_helper.rb

```diff
@@ -68,8 +67,4 @@
   config.before(:each) do |example|
     ApplicationController.handle_exceptions = false if %w(controller requests).include?(example.metadata[:type])
   end
-
-  config.around(:each) do |example|
-    EvmSpecHelper.clear_caches { example.run }
-  end
```

Comment:
For my own info, what is this for exactly? I'm assuming only for something outside of this repo.

Reply:
This is already done by manageiq's spec_helper.  It clears all things that are `cache_with_timeout` and a few other classes that have a cache that may need to be reset between tests.